rpm/kubelet.spec

```diff
@@ -52,11 +51,11 @@ Command-line utility for interacting with a Kubernetes cluster.
 
 %package -n kubeadm
 
-Version: 1.6.0
-Release: %{RPM_RELEASE}.alpha.0.2074.a092d8e0f95f52
-Summary: Command-line utility for administering a Kubernetes cluster. (ALPHA)
-Requires: kubelet >= 1.4.0
-Requires: kubectl >= 1.4.0
+Version: %{KUBE_VERSION}
+Release: %{RPM_RELEASE}
+Summary: Command-line utility for administering a Kubernetes cluster.
+Requires: kubelet >= 1.6.0
+Requires: kubectl >= 1.6.0
```

Comment:
Could probably re-use KUBE_VERSION here if you wanted, but in any case everything here looks ok to me.